schema/PharmGKBIDs.mcf

```diff
@@ -0,0 +1,176 @@
+# generated by script based on dictionaries from
+# /scripts/pharmgkb/drug_gene_relations/config.py , then modified by hand
```

Comment:
Can you expand the note a bit more to say how modified by hand?

Reply:
+1

Reply:
Thanks!